lib/shoryuken/environment_loader.rb

```diff
@@ -79,6 +79,10 @@ def load_rails
       end
     end
 
+    def load_rails?
+      options[:rails] || Shoryuken.options[:rails]
```

Comment:
Is it possible to rely solely on the CLI arg `options[:rails]` here? It's not clear to me in what situation `options[:rails]` would be falsey here, but `Shoryuken.options[:rails]` would be truthy here, especially since by the time this method is invoked, `initialize_options` has not yet run.

Reply:
I would be ok with just having `options[:rails]`, that is how I'm using it.  Though, I wanted to keep the existing functionality in case someone was enabling it through the `config/shoryuken.yml` file.

Reply:
Correct me if I'm wrong, but `Shoryuken.options[:rails]` won't be populated by the value in `config/shoryuken.yml` until `initialize_options` runs. So if the intent is to preserve functionality of setting `rails: true` in the config file, I don't think this code accomplishes that.

Either way, I don't think it's necessary to preserve the behavior of allowing `rails: true` to be set in the config file because it's a bit circular: if we have to read the config file to know whether or not to load rails, but we also don't want to read the config file until _after_ initializing Rails, that seems messy.

Reply:
That's right, by the time `initialize_options` runs, it has already passed the option to `initialize_rails`.

> if we have to read the config file to know whether or not to load rails, but we also don't want to read the config file until after initializing Rails, that seems messy.

That is a very good point!

Moving forward with this change to only allowing CLI arg to load rails.